client/wordpress-com.js

```diff
@@ -188,6 +188,13 @@ const sections = [
 		secondary: true,
 		group: 'checklist',
 	},
+	{
+		name: 'google-my-business',
+		paths: [ '/google-my-business' ],
+		module: 'my-sites/google-my-business',
+		secondary: true,
+		group: 'google-my-business',
```

Comment:
I wonder if we shouldn't use `sites` instead of `google-my-business` for `group`.

Reply:
I took an example from `checklist` above, are you suggesting it _should_ be `sites` ?

Reply:
Exactly. I can't remember off the top of my head what this `group` parameter is for, but we should check that and makes sure we're using the right value.

Reply:
Indeed we probably want to switch to `sites` here.
Group is used for customizing the site title in case we're on a specific site section it seems.
See https://github.com/Automattic/wp-calypso/blob/master/client/state/ui/selectors.js#L86 and https://github.com/Automattic/wp-calypso/blob/master/client/state/document-head/selectors.js#L75